.eslintrc.js

```diff
@@ -22,10 +22,9 @@ module.exports = {
   ],
   parserOptions: {
     sourceType: 'module',
-    ecmaVersion: 7,
+    ecmaVersion: 2018,
```

Comment:
Can you check the version on our `core-js` polyfill to ensure that we are polyfilling everything we can for ECMA2018?
Note that there is on regex feature that cannot be polyfilled, see here: http://kangax.github.io/compat-table/es2016plus/

Reply:
Core-js is at 2.5.3, which appears to polyfill a few ES2018 feature used in the code, like Promise.finally. There is a v3 for core-js, though.

Reply:
Is the main requirement that we fail loudly if someone uses a feature that's not polyfilled? (To avoid 'it worked on my machine'-style bugs)

Reply:
Yeah, does eslint have individual linting rules for ECMA2018 features? (if not, we might have to write a custom one, a regex for the unsupported regex feature!)